extensions/markdown-math/src/extension.ts

```diff
@@ -6,16 +6,21 @@ import * as vscode from 'vscode';
 
 declare function require(path: string): any;
 
-const enabledSetting = 'markdown.math.enabled';
+const markdownMathSetting = 'markdown.math';
 
 export function activate(context: vscode.ExtensionContext) {
 	function isEnabled(): boolean {
 		const config = vscode.workspace.getConfiguration('markdown');
 		return config.get<boolean>('math.enabled', true);
 	}
 
+	function getMacros(): { [key: string]: string } {
+		const config = vscode.workspace.getConfiguration('markdown');
+		return config.get<{ [key: string]: string }>('math.macros', {});
+	}
+
 	vscode.workspace.onDidChangeConfiguration(e => {
-		if (e.affectsConfiguration(enabledSetting)) {
+		if (e.affectsConfiguration(markdownMathSetting)) {
```

Comment:
Sanity check: does this change make it so that the plugins are reloaded if any setting key starting with `markdown.math` is changed?

Reply:
Yes. With that change, whenever a setting key starting with `markdown.math` is modified, the plugins related to markdown are reloaded.